### ISSUE TITLE
TailBag not working

### ISSUE THREAD

**Issue by twixarr** [naelstrof/KoboldKare#38]
the tailbag item hasn't allowed multi-pickup since the last major update

**Comment by naelstrof** [comment]
Yeah sorry, this is intentional. Functionally might change later, but right now grab upgrades are done through eating beef and burgers.
